packages/poi-preset-transform-test-files/index.js

```diff
@@ -2,27 +2,37 @@ const path = require('path')
 const globby = require('globby')
 const webpack = require('webpack')
 
-module.exports = () => {
+module.exports = (options = {}) => {
+  const { outputDir = '' } = options
+
   return poi => {
     poi.extendWebpack('test', config => {
-      const outputDir = path.resolve(poi.options.cwd, 'output_test')
+      const outputPath = path.resolve(poi.options.cwd, outputDir)
 
-      config.output.path(outputDir)
+      config.output.path(outputPath)
+      config.output.filename('[name].transformed.js')
 
       // Exclude node_modules in bundle file
       poi.webpackUtils.externalize(config)
     })
 
     poi.run('test', webpackConfig => {
+      const baseDir = poi.argv.baseDir || poi.options.cwd
       const input = poi.argv._.slice(1)
-      const inputFiles = input.length > 0 ? input : ['**/*.test.js']
+      const inputFiles = input.length > 0 ? input : ['**/*.{test,spec}.js']
       const ignores = ['!**/node_modules/**', '!**/vendor/**']
 
-      return globby(inputFiles.concat(ignores), { cwd: poi.options.cwd })
+      return globby(inputFiles.concat(ignores), { cwd: baseDir })
         .then(files => {
           delete webpackConfig.entry.client
-          webpackConfig.entry.test = files
-            .map(file => path.resolve(poi.options.cwd, file))
+          webpackConfig.entry = files.reduce((acc, file) => {
+            const filename = outputDir ? path.basename(file) : file
```

Comment:
why `path.basename`? seems `test/nest/test.js` and `test/test.js` will end up being transformed to the same location which is `outputDir/test.transformed.js`

Reply:
In my test, the location would be different as default when `outputDir` is empty.

The spec is: 
If users not specify `outputDir`, the `test/nest/test.js` and `test/test.js` will transform as `test/nest/test. transformed.js` and `test/test.transformed.js`. However, if `outputDir` is not '', they would be put into `outputDir/...` centrally. 

The reason is that Jest encourage developers put test file around source file under `__test__`, which is dispersed. In order to match the pattern, I set in place transform as default behaviour.